ferrocene/doc/symbol-report.json

```diff
@@ -4121,10 +4121,6 @@
   "core::ptr::alignment::Alignment::new_unchecked",
   "core::ptr::alignment::Alignment::new_unchecked::precondition_check",
   "core::ptr::const_ptr::<impl *const T>::add",
-  "core::ptr::const_ptr::<impl *const T>::add::precondition_check",
```

Comment:
This will fail in CI since these functions are present in `ferrocene-dist` profile. Please revert this or run the update with that profile.